tests/commands/help.rb

```diff
@@ -8,25 +8,29 @@ def test_no_arguments
       Available commands: build, check, const, deps, download, files, help, install, license, list, postinstall, prop, reinstall, remove, search, sysinfo, test, update, upgrade, upload, version, whatprovides
```

Comment:
I think this list could be calculated too.

Reply:
We add new commands even less frequently than we add new properties-- the case for doing this is weaker than the case for doing it for properties.

Reply:
Not weak if it helps make sure tests don't fail when new properties are added and this test update was missed.

Reply:
That's the whole point of tests-- to make sure we don't miss the updates.

Reply:
We can address this on a future PR.

Reply:
Replace with?
```
Available commands: #{CREW_COMMANDS}
```